vortex-vector/src/listview/vector_mut.rs

```diff
@@ -300,27 +301,23 @@ impl VectorMutOps for ListViewVectorMut {
         self.len = self.validity.len();
     }
 
-    /// This will also panic if we try to extend the `ListViewVector` beyond the maximum offset
-    /// representable by the type of the `offsets` primitive vector.
     fn extend_from_vector(&mut self, other: &ListViewVector) {
```

Comment:
I wonder if we want to be able to specify the max size to not cast from u8 -> u16 -> u32...

Reply:
How we don't know the offset of size max value? We can only guess based on avg list len